src/DependencyInjection/NeustaConverterExtension.php

```diff
@@ -67,6 +74,63 @@ private function registerConverterConfiguration(string $id, array $config, Conta
             ]);
     }
 
+    /**
+     * @param array<string, mixed> $config
+     */
+    private function registerPopulatorConfiguration(string $id, array $config, ContainerBuilder $container): void
+    {
+        $arguments = [];
+        if (empty($config['class']) || ConvertingPopulator::class === $config['class']) {
+            $arguments = $this->buildArgumentsForConvertingPopulator($config);
+        } elseif (ArrayConvertingPopulator::class === $config['class']) {
+            $arguments = $this->buildArgumentsForArrayConvertingPopulator($config);
+        }
```

Comment:
This is a good case for the new `match` expression:

```suggestion
        $arguments = match ($config['class']) {
            ConvertingPopulator::class => $this->buildArgumentsForConvertingPopulator($config),
            ArrayConvertingPopulator::class => $this->buildArgumentsForArrayConvertingPopulator($config),
            default => [],
        };
```

(also add `->cannotBeEmpty()` to the `class` config node)

Reply:
See #55